nestkernel/conn_builder.cpp

```diff
@@ -337,6 +337,61 @@ nest::ConnBuilder::single_connect_( index sgid,
   }
 }
 
+inline void
+nest::ConnBuilder::skip_conn_parameter_( thread target_thread, librandom::RngPtr& rng )
```

Comment:
The skip()-function is now simplified such that only parameters specified as arrays require skipping. This is accomplished by collecting all pointers to array parameter in a vector after their initialisation.